### PR TITLE
Update rdm.md

### LINKED ISSUE #1883: Add an image in the **Research Data Management** page
- **Problem:** The Turing Way chapters are better explained with illustrations and images. We have many chapters that are not connected with any illustration yet.

- **Solution:** Below we provide chapters with the details on the image with links to the chapter and image file in The Turing Way. Please edit the chapter file to add images.

- **How-to add images in The Turing Way:** We use the following 'directive' in the chapter file to add different images (text starting and should also end with ``` ). Where <FILE-PATH>) is the location of the image file, <FILE-NAME> is the name of the file, and <IMAGE-DESCRIPTION>) is used to describe the image for screenreading apps.

```
```{figure} <FILE-PATH>
---
name: <FILE-NAME>
alt: <IMAGE-DESCRIPTION>
---
_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. ```
```
``

*You can find more details in [The Turing Way Style Guide](https://the-turing-way.netlify.app/community-handbook/style/style-figures.html#alternative-text)*

**Task**
- Visit the chapter file (link provided)
- Start editing (click on the pencil symbol) 
- Copy-past the 'directive' to add images
- Replace the <FILE-PATH> and <FILE-NAME>
- Provide details for "IMAGE-DESCRIPTION" by visiting the image link provided for each challenge.

- Add an image in the **Research Data Management** page
    * Chapter: "Research Data Management": https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/reproducible-research/reproducible-research/rdm.md (click the link to directly edit the file - no need to fork separately) 
    * FILE-PATH: ../figures/DataEcosystem.jpg
    * FILE-NAME: DataEcosystem
    * Provide details for "IMAGE-DESCRIPTION" by visiting the image [here](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/website/figures/DataEcosystem.jpg).

### PR DESCRIPTION
**Pull request by jcolomb**
answer to https://github.com/alan-turing-institute/the-turing-way/issues/1883

### Summary

Fixes #1883

link to the figure not working, I do not understand what to do.

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### List of changes proposed in this PR (pull-request)

* *Lorem ipsum dolor sit amet, consectetur adipiscing.*
* *Lorem ipsum dolor sit amet, consectetur adipiscing.*

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [X] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/reproducible-research/rdm.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+```{figure} ../figures/DataEcosystem.jpg
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm.md:1]:
```suggestion
```{figure} ../../figures/data-ecosystem.jpg
```

File: book/website/reproducible-research/rdm.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+```{figure} ../../figures/data-ecosystem.jpg
+---
+name: data-ecosystem
+alt: image of the data ecosystem with private and public data
+---
+_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. 
+```
+
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm.md:8]:
```suggestion

```{figure} ../../figures/data-ecosystem.jpg
---
height: 400px
name: data-ecosystem
alt: image of the data ecosystem with private and public data
---
Open and closed data for reproducibility.
_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. 
```

```

File: book/website/reproducible-research/rdm.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+```{figure} ../../figures/data-ecosystem.jpg
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm.md:1]:
```suggestion
```{figure} ../../figures/data-ecosystem.jpg
```

File: book/website/reproducible-research/rdm.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
+---
+name: data-ecosystem
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm.md:3]:
```suggestion
height: 400px
name: data-ecosystem
```

File: book/website/reproducible-research/rdm.md [hunk 5]
```diff
@@ -1,3 +1,11 @@
...
+alt: image of the data ecosystem with private and public data
+---
+_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300.
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm.md:6]:
```suggestion
Open and closed data for reproducibility.
_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. 
```

File: book/website/reproducible-research/rdm.md [hunk 6]
```diff
@@ -1,3 +1,13 @@
+```{figure} ../../figures/data-ecosystem.jpg
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm.md:1]:
```suggestion
```{figure} ../../figures/DataEcosystem.jpg
```

**Comment by jcolomb** [comment]
figures won't show, tried different links.

File: book/website/reproducible-research/rdm.md [hunk 7]
```diff
@@ -1,3 +1,13 @@
+```{figure} ../../figures/DataEcosystem.jpg
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm.md:1]:
```suggestion
```{figure} ../figures/Data-Ecosystem.jpg
```

**Comment by malvikasharan** [comment]
Thank you for the PR @jcolomb. I have updated the image file and it seems fine now. Let me know if the issue you were facing is solved with the change.

**Review by malvikasharan — APPROVED** [review]

**Comment by malvikasharan** [comment]
Also, nice to see this new diff feature for images. 👯 
image

**Comment by jcolomb** [comment]
ready to merge!

**Pull request merged**